cluster/gce/container-linux/configure-helper.sh

```diff
@@ -34,6 +34,22 @@ function create-dirs {
   fi
 }
 
+# Vars assumed:
```

Comment:
Currently I've got this logic duplicated here and in gci/configure-helper.sh.

Not sure where the best place to commonize this would be, but happy to take suggestions.